keyboards/pimoroni/keybow2040/keymaps/via/keymap.json

```diff
@@ -0,0 +1,42 @@
+{
+    "keyboard": "pimoroni/keybow2040",
+    "author": "fanf2",
+    "version": 1,
+    "config": {
+        "features": {
+            "via": true
+        }
+    },
+    "keymap": "via",
+    "layout": "LAYOUT",
+    "layers": [
+	[
+            "KC_A",    "KC_B",    "KC_C",    "KC_D",
+            "KC_E",    "KC_F",    "KC_G",    "KC_H",
+            "KC_I",    "KC_J",    "KC_K",    "KC_L",
+            "KC_M",    "KC_N",    "KC_O",    "KC_P",
+	    "KC_Q"
+	],
+	[
+            "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS",
+            "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS",
+            "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS",
+            "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS",
+            "KC_TRNS"
+	],
+	[
+            "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS",
+            "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS",
+            "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS",
+            "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS",
+            "KC_TRNS"
+	],
+	[
+            "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS",
+            "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS",
+            "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS",
+            "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS",
+            "KC_TRNS"
+	]
```

Comment:
Padding the VIA keymap to the maximum number of layers is no longer needed — recent QMK can detect the number of actually initialized layers and fill any uninitialized layers with `KC_TRNS` automatically.
```suggestion
```

Reply:
Cleaned up in round two